wsagent/che-core-git-impl-jgit/src/main/java/org/eclipse/che/git/impl/jgit/JGitConnection.java

```diff
@@ -1605,7 +1608,7 @@ public void configure(Transport transport) {
             }
 
             ProxyAuthenticator.initAuthenticator(remoteUrl);
-
+            UserAgent.set(USER_AGENT);
```

Comment:
do we need to call this static method from JGitConnection instance or from a class that is less created like JGitConnectionFactory ?